include/uv.h

```diff
@@ -236,6 +236,7 @@ typedef struct uv_interface_address_s uv_interface_address_t;
 typedef struct uv_dirent_s uv_dirent_t;
 typedef struct uv_passwd_s uv_passwd_t;
 typedef struct uv_utsname_s uv_utsname_t;
+typedef struct uv_credentials_s uv_credentials_t;
```

Comment:
Like's bikeshed away. :)

I wouldn't mind adding `pipe_` but not sure if it should be shortened from `credentials` to `creds` - otherwise, I think the API function should also be shortened to `uv_pipe_get_creds()`.

Reply:
> I wouldn't mind adding `pipe_` but not sure if it should be shortened from `credentials` to `creds` - otherwise, I think the API function should also be shortened to `uv_pipe_get_creds()`.

Makes sense to me!